.github/workflows/fork-sync.yml

```diff
@@ -0,0 +1,40 @@
+on:
+  schedule:
+    - cron:  '0 7 * * 1,4'
+  workflow_dispatch:
```

Comment:
Add inputs to match version specified in the StirlingLabs/XKCP workflow.

Example input:
```json
{
    "event_type": "New XKCP Release",
    "client_payload": { "tag": "v21.07.0" }
}
```

Reference: [GitHub Actions: Manual triggers with workflow_dispatch [2020/07/06]](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/)